modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Toggle.scss

```diff
@@ -1,5 +1,10 @@
 @import 'atlas-variables';
 
-.lfr-objects__toggle-tooltip-icon {
-	color: $secondary-l1;
+.lfr-objects__toggle {
+	display: flex;
+	flex-direction: row;
```

Comment:
Hi @Artartur, as the default value of flex-direction is row, could you check if this row is really necessary?

Reply:
You are right, I've changed it, thx for the review @igor-franca